model/trace/messaging.yaml

```diff
@@ -117,6 +117,13 @@ groups:
       - ref: messaging.rabbitmq.destination.routing_key
         requirement_level:
           conditionally_required: If not empty.
+      - id: delivery_tag
```

Comment:
The messaging attributes have been moved to the registry, so this one needs to be added there now: https://github.com/open-telemetry/semantic-conventions/blob/main/model/registry/messaging.yaml

Then here, you just use `ref` as the rest of attributes. And re-generate the markdown.

Reply:
I'll take care of it :)

Reply:
fixed